pyiron/lammps/interactive.py

```diff
@@ -539,7 +519,29 @@ def update_potential(self):
         self._interactive_lib_command(self.potential.Config[0][1])
 
     def interactive_indices_getter(self):
-        return super(LammpsInteractive, self).interactive_indices_getter().tolist()
+        lammps_indices = np.array(self._interactive_library.gather_atoms("type", 0, 1))
+        indices = self.remap_indices(lammps_indices)
+        return indices.tolist()
+
+    def interactive_indices_setter(self, indices):
```

Comment:
Just for my understanding, this function was only moved, but nothing changes, right?

Reply:
Yes, just to make the getter and setter code adjacent; they were previously quite far apart.